lightwood/encoder/categorical/autoencoder.py

```diff
@@ -101,12 +100,11 @@ def encode(self, column_data):
                 embeddings = self.encoder(oh_encoded_tensor)
                 return embeddings.to('cpu')
 
-    def decode(self, encoded_data):
-        self.onehot_encoder.predict_proba = self.predict_proba
+    def decode(self, encoded_data, predict_proba=False):
```

Comment:
!!IMPORATN!! The auto encoder doesn't need to do this period, we never use autoencoders for targets, wouldn't make much sense